pkgs/development/python-modules/scrapy/default.nix

```diff
@@ -1,35 +1,8 @@
-{ lib
-, stdenv
-, botocore
-, buildPythonPackage
-, cryptography
-, cssselect
-, fetchPypi
-, fetchpatch
-, glibcLocales
-, installShellFiles
-, itemadapter
-, itemloaders
-, jmespath
-, lxml
-, packaging
-, parsel
-, pexpect
-, protego
-, pydispatcher
-, pyopenssl
-, pytestCheckHook
-, pythonOlder
-, queuelib
-, service-identity
-, setuptools
-, sybil
-, testfixtures
-, tldextract
-, twisted
-, w3lib
-, zope-interface
-}:
+{ lib, stdenv, botocore, buildPythonPackage, cryptography, cssselect, fetchPypi
+, glibcLocales, installShellFiles, itemadapter, itemloaders, jmespath, lxml
+, fetchpatch, packaging, parsel, pexpect, protego, pydispatcher, pyopenssl
+, pytestCheckHook, pythonOlder, queuelib, service-identity, setuptools, sybil
+, testfixtures, tldextract, twisted, w3lib, zope-interface }:
```

Comment:
Please keep it one entry per line. Multiple entries per per line can't be sorted in a easy way and reviewing is harder.

Reply:
Thanks, we'll keep this in mind, nixfmt just did what are its current default at the moment.